assets/js/map.js

```diff
@@ -0,0 +1,39 @@
+var greenIcon = new L.Icon({
+  iconUrl: "/assets/img/icons/icon-map-pin.png",
+})
+
+const displayMap = (lat, lng) => {
+  const map = L.map('map', {zoomControl: false}).setView([lat, lng], 11)
+  .on("locationfound", e => {
+    console.log(e);
+  });
+  L.marker([lat, lng], { icon: greenIcon }).addTo(map);
+  L.tileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png', {
+    maxZoom: 19,
+    attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
+  }).addTo(map);
+}
+
+const getUserLoaction = () => {
```

Comment:
Hey, @ALFAROSO  check this out to get the current position. I think you might use it.